packages/webpack/lib/plugins/log.js

```diff
@@ -47,22 +58,33 @@ exports.LoggerPlugin = class LoggerPlugin {
       if (this.lastHashes[name] === stats.hash) {
         return;
       }
+      const { hints, maxAssetSize } = this.performance;
       const { hash, startTime, endTime } = stats;
       const isRebuild = this.lastHashes[name];
       const duration = prettyMS(endTime - startTime);
+      const { assets } = stats.toJson(statsToJsonOptions);
+      const emitAssetSizeFeedback =
```

Comment:
Variable naming appears to be a tad inconsistent here: `isRebuild` vs. `emitAssetSizeFeedback`... Should it not be `shouldEmit...`?